src/index.js

```diff
@@ -1,7 +1,114 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import L from 'leaflet';
+ReactDOM.render(	
```

Comment:
This render shouldn't be here. Not only does this negatively impact the readability score, but you already have a render function on the bottom of your file. In addition, the DOM element it is targeting technically does not exist in the HTML document. See my comment in your index.html file for more details on that.